reactor-core/src/test/java/reactor/core/publisher/FluxSwitchOnFirstTest.java

```diff
@@ -98,6 +101,71 @@ public void shouldNotSubscribeTwiceConditional() {
                   .hasMessage("FluxSwitchOnFirst allows only one Subscriber");
     }
 
+    @Test
+    public void shouldNotSubscribeTwiceWhenCanceled() {
+        Schedulers.single().start();
```

Comment:
Could you please explain why we need to start it here?

Reply:
Unless the `Scheduler` was disposed, which should be a NO-OP since it is one of the default `Schedulers`, `start()` is not needed indeed.

Reply:
fixed